api/include/awa/static.h

```diff
@@ -513,20 +568,20 @@ AwaError AwaStaticClient_CreateObjectInstance(AwaStaticClient * client, AwaObjec
  * @param[in] resourceID The ID of the resource to mark as changed.
  * @return AwaError_Success if the specified resource exists and was successfully marked as changed.
  * @return AwaError_DefinitionInvalid if no resource exists for the specified object, object instance and resource IDs.
- * @return AwaError_StaticClientInvalid if the specified client is NULL or uninitialised.
+ * @return AwaError_StaticClientInvalid if @e client is NULL or uninitialised.
  */
 AwaError AwaStaticClient_ResourceChanged(AwaStaticClient * client, AwaObjectID objectID, AwaObjectInstanceID objectInstanceID, AwaResourceID resourceID);
 
 /**
- * @brief Mark the specified object instance as changed, in order for the Awa Client Daemon to
- *        send notifications to any LWM2M servers observing that object instance.
+ * @brief Mark the specified object instance as changed, in order for the Awa Static Client to
+ *        send notifications to all LWM2M servers observing that object instance.
  *
```

Comment:
Maybe we should add a note that states this should be called when a resource is created or deleted on an object instance.